assets/js/view/part/towns.rb

```diff
@@ -17,7 +17,10 @@ class Towns < Snabberb::Component
       def render
         @tile.towns.map do |town|
           if @tile.lawson?
-            h(Part::TownDot, region_use: @region_use, color: color_for(town))
+            h(Part::TownDot, tile: @tile, region_use: @region_use, color: color_for(town))
+          elsif @tile.paths.empty?
+            # This town is all alone, no routes to it
+            h(Part::TownDot, tile: @tile, region_use: @region_use, color: color_for(town))
```

Comment:
this looks exactly the same, how about tile.lawson? || empty

Reply:
of course, lol wasn't the same during debugging lol